dataflow/src/domain/mod.rs

```diff
@@ -30,6 +34,7 @@ pub struct Config {
 }
 
 const BATCH_SIZE: usize = 256;
+const RECOVERY_BATCH_SIZE: usize = 512;
```

Comment:
Why not larger? We know nothing else is happening in the graph until it has recovered anyway.

Reply:
@fintelia suggested 512 - I'm not completely sure what the trade-offs are here. What do you think would be a good number?